thewalrus/tests/test_quantum.py

```diff
@@ -1813,3 +1814,32 @@ def test_n_body_marginals_too_high_correlation():
         ValueError, match="The order of the correlations is higher than the number of modes"
     ):
         n_body_marginals(mu, cov, 4, 4)
+
+
+
+def test_click_cumulants():
+    """Tests the correctness of the click cumulant function"""
+    M = 3
+    cov = random_covariance(M)
+    mu = np.zeros([2 * M])
+    mu0, cov0 = reduced_gaussian(mu, cov, [0])
+    mu1, cov1 = reduced_gaussian(mu, cov, [1])
+    mu2, cov2 = reduced_gaussian(mu, cov, [2])
+    mu01, cov01 = reduced_gaussian(mu, cov, [0, 1])
+    mu02, cov02 = reduced_gaussian(mu, cov, [0, 2])
+    mu12, cov12 = reduced_gaussian(mu, cov, [1, 2])
+    expected = (
+        threshold_detection_prob_displacement(mu, cov, [1, 1, 1])

```

Comment:
if you use `threshold_detection_prob` here instead of `threshold_detection_prob_displacement`, this will make numba happier, as it will never be given python lists as args.

Reply:
Great point! Done.